.github/workflows/automerge.yaml

```
@@ -103,7 +103,7 @@ jobs:
 
       - id: automerge
         name: Auto merge of cherry-picked PRs.
-        uses: "pascalgn/automerge-action@v0.15.5"
+        uses: "pascalgn/automerge-action@v0.16.2"
```

Quotes can be removed here, I think
```suggestion
        uses: pascalgn/automerge-action@v0.16.2
```